readme.md

```diff
@@ -109,21 +109,26 @@ $ npm -v
   console.log(profile)
   ```
 
-* **Question:** What gets logged? Why?
+* **Question:** What gets logged? Why? 
+
+    #. 'shawn'  
+    #. 'hello node'
```

Comment:
The question here is asking why as well. `'shawn'` gets logged because you've exported it from your other file.